core/src/test/scala/GraphTests.scala

```diff
@@ -167,20 +167,16 @@ object GraphTests extends Properties("Graph") {
       s"Paths different: cheapest = $cPath, shortest = $sPath" |: sPath === cPath
     }
 
-  {
-    import GDecomp._
-    import org.scalacheck.ScalacheckShapeless._
+  import GDecomp._
+  import org.scalacheck.ScalacheckShapeless._
 
-    // TODO Clean these up, probably into a GraphCogen class. Keep the questionable ones hidden.
+  // TODO Clean these up, probably into a GraphCogen class. Keep the questionable ones hidden.
+  implicit def cogenMap[K: Cogen: Ordering, V: Cogen]: Cogen[Map[K, V]] =
+    Cogen.it(_.toVector.sortBy(_._1).iterator)
+  implicit def cogenContext[V: Cogen: Ordering, A: Cogen, B: Cogen: Ordering]: Cogen[Context[V, A, B]] =
+    Cogen[(V, GrContext[V, A, B])].contramap(c => c.vertex -> c.toGrContext)
```

Comment:
These are falsely getting flagged as unused in certain versions of Scala.  I believe they will be unnecessary after a new release of scalacheck.

This is sloppy, but since it doesn't affect the public API, I'm content to live with it.

Reply:
`cogenMap` won't be necessary, but `cogenContext` will be because of the no-ordering assumption